build.gradle

```diff
@@ -154,7 +155,8 @@ task printVersion {
 configure(subprojects.findAll {it.name != 'jaeger-thrift'}) {
    apply plugin: 'net.ltgt.errorprone'
    dependencies {
-     errorprone 'com.google.errorprone:error_prone_core:2.0.15'
+     // stay on 2.0.5, 2.1.1 throws on analysing
+     errorprone 'com.google.errorprone:error_prone_core:2.0.5'
```

Comment:
We shouldn't be downgrading `error_prone` because we might be losing some of the more recent checks.

Reply:
I agree, .15 -> .5 was a typo